Gruntfile.js

```diff
@@ -252,12 +265,126 @@ module.exports = function (grunt) {
+    'electron-installer-debian': {
+      options: {
+        productName: LINUX_APPNAME,
+        productDescription: 'Run containers through a simple, yet powerful graphical user interface.',
+        section: 'devel',
+        priority: 'optional',
+        icon: './util/kitematic.png',
+        lintianOverrides: [
+          'changelog-file-missing-in-native-package',
+          'executable-not-elf-or-script',
+          'extra-license-file'
+        ],
+        categories: [
+          'Utility'
+        ],
+        rename: function (dest, src) {
+          return dest + '<%= name %>_'+ packagejson.version +'-<%= revision %>_<%= arch %>.deb';
+        }
+      },
+      linux64: {
+        options: {
+          arch: 'amd64'
+        },
+        src: './dist/Kitematic-linux-x64/',
+        dest: './dist/',
+      },
+      linux32: {
+        options: {
+          arch: 'i386'
+        },
+        src: './dist/Kitematic-linux-ia32/',
+        dest: './dist/',
+      }      
+    },
+    'electron-installer-redhat': {
+      options: {
+        productName: LINUX_APPNAME,
+        productDescription: 'Run containers through a simple, yet powerful graphical user interface.',
+        priority: 'optional',
+        icon: './util/kitematic.png',        
+        categories: [
+          'Utilities'
+        ],
+        rename: function (dest, src) {
+          return dest + '<%= name %>_'+ packagejson.version +'-<%= revision %>_<%= arch %>.rpm';
+        }
+      },
+      linux64: {
+        options: {
+          arch: 'x86_64'
+        },
+        src: './dist/Kitematic-linux-x64/',
+        dest: './dist/'
+      },
+      linux32: {
+        options: {
+          arch: 'x86'
+        },
+        src: './dist/Kitematic-linux-ia32/',
+        dest: './dist/'
+      }
     }
   });
 
+  // Load the plugins for linux packaging
+  grunt.loadNpmTasks('grunt-electron-packager');
+  grunt.loadNpmTasks('grunt-electron-installer-debian');
+  grunt.loadNpmTasks('grunt-electron-installer-redhat');
+
   grunt.registerTask('default', ['newer:babel', 'less', 'newer:copy:dev', 'shell:electron', 'watchChokidar']);
-  if (!IS_WINDOWS) {
-    grunt.registerTask('release', ['clean:release', 'babel', 'less', 'copy:dev', 'electron', 'copy:osx', 'shell:sign', 'shell:zip', 'copy:windows', 'rcedit:exes', 'compress']);
+
+  if (!IS_WINDOWS && !IS_LINUX) {
+    grunt.registerTask('release', ['clean:release', 'babel', 'less', 'copy:dev', 'electron', 'copy:osx', 'shell:sign', 'shell:zip', 'copy:windows', 'rcedit:exes', 'compress', 'shell:linux_npm', 'electron-packager:osxlnx', 'electron-installer-debian:linux64']);
+  }else if(IS_LINUX){
+
+    var linuxpackage = null;
+    // linux package detection
+    if(IS_DEB && IS_X64){
+      linuxpackage = 'electron-installer-debian:linux64'; 
+    }else if(IS_DEB && IS_I386){
+      linuxpackage = 'electron-installer-debian:linux32';
+    }else if(IS_RPM && IS_X64){
+       linuxpackage = 'electron-installer-redhat:linux64'; 
```

Comment:
fix indentation for 6-spaces not 7.

if you use atom to edit things, get the `linter` + `linter-eslint` packages